phabricatoremails/error_notify.py

```diff
@@ -0,0 +1,24 @@
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import traceback
+from dataclasses import dataclass
+from typing import Any
+
+import sentry_sdk
+from statsd import StatsClient
+
+
+@dataclass
+class ErrorNotify:
+    _logger: Any
+    _stats: StatsClient
+
+    def notify(self, exception: Exception, warning: str, failure_stat: str):
```

Comment:
Can you add a docstring here describing what this method does? Something like "Log formatted exceptions, increment stats, and capture exceptions in Sentry."